src/components/autocomplete/js/highlightDirective.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+  'use strict';
+  angular
+      .module('material.components.autocomplete')
+      .directive('mdHighlightText', MdHighlight);
+
+  function MdHighlight () {
+    return {
+      terminal: true,
+      controller: 'MdHighlightCtrl',
+      controllerAs: 'mdCtrl'
```

Comment:
Since `MdHighlightCtrl` does not expose any public API and since there is no template, what is the purpose of `controllerAs` ? If it is not used then it just creates an unneeded object reference on the scope.